tests/test_extraction.py

```diff
@@ -1,31 +1,208 @@
-import unittest
-from textextraction import doc_process_toolkit as dpt
+import os
+from unittest import TestCase, skipUnless, main
+from textextraction.textextraction import (TextExtraction, PDFTextExtraction,
+                                           textextractor)
 
+ALL_INSTALLED = os.getenv("ALL_INSTALLED")
```

Comment:
It's unfortunate, but I think for these tests to run you need to have installed everything. That's painful and not the spirit of unit tests - but such is the code.